todo.py

```diff
@@ -1,21 +1,62 @@
 
-def print_todos(todos):
-    if len(todos) == 0:
+pending_list = []
```

Comment:
Using two lists is a solid approach, but it may not be necessary once you have dictionaries. If each todo is a dictionary, and each has a `completed` property, could you use the `completed` property to help you manage your todos? (i.e., print only pending todos based on the value of the `completed` property)